main.py

```diff
@@ -66,6 +66,28 @@ def validate_google_user(provided_id_token):
 		return False
 	return True
 
+
+@app.route('/user/registration', methods=['POST'])
+def handle_registration():
+	if request.method == 'POST':
+		content = request.get_json()
+		# grabs values from JSON sent from mobile application
+		provided_gid = content.get("gid")
+		provided_name = content.get("displayName")
+		provided_avatar = content.get("avatar")
+
+		user_status = {"addSuccessful": True}
+
+		# creating a new user to register into database
+		new_user = datastore.entity.Entity(key=datastore_client.key(constants.userEntity))
+		new_user.update({"displayName": provided_name, "gid": provided_gid, "avatar": provided_avatar})
+		datastore_client.put(new_user)
+		user_status = {"loginSuccessful": True}
+		return json.dumps(user_status), 201
+	else:
+		return "Method not recognized"
```

Comment:
Will this return a 200 or a 4xx/5xx HTTP exception?

Reply:
From our last meeting - unless we want to set up the layout for accepting additional methods in the future, we can remove the if-then statement since only POST methods are accepted (b/c line 70). I'll let you decide on that.